docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -70,6 +70,8 @@
     * [SolrJ](https://github.com/apache/lucene-solr/tree/master/solr/solrj) 7.x
   * [Cassandra](https://github.com/apache/cassandra) 3.x
     * [cassandra-java-driver](https://github.com/datastax/java-driver) 3.7.0-3.7.2
+  * Hbase
+    * [hbase-client](https://github.com/apache/hbase) HTable 1.x
```

Comment:
HBase client with HTable? What is the relationship between these?

Reply:
The HBase Client also has classes that interact with the remote server, such as `Admin `, which can modify tables, operate clusters, manage connections, etc. The PR only monitors the HTABLE